wallet/src/main/java/com/android/identity_credential/wallet/SelfSignedMdlIssuingAuthority.kt

```diff
@@ -77,6 +77,9 @@ class SelfSignedMdlIssuingAuthority(
     override fun getProofingGraphRoot(
         registrationResponse: RegistrationResponse,
     ): SimpleIssuingAuthorityProofingGraph.Node {
+        val devAssets = mapOf("tools.svg" to resourceBytes(R.raw.tools))
```

Comment:
I think we should just use the existing icon for devmode that we use in Settings and elsewhere (`R.drawable.experiment_icon`).